pillar/components/sspl.sls

```diff
@@ -96,12 +96,6 @@ sspl:
     threaded: "true"
     logging_interval: "3600"
   STORAGE_ENCLOSURE:
-    primary_controller_ip: "127.0.0.1"
```

Comment:
https://github.com/Seagate/cortx-prvsnr/pull/583#discussion_r499727186

same comment applies. So what else is the cleanup candidate here ?

Reply:
@shubhamseagate 
You might want to remove:
```
  IPMI:
    user: "admin"
    pass: "admin"
  RABBITMQINGRESSPROCESSOR:
    virtual_host: "SSPL"
    queue_name: "actuator-req-queue"
    exchange_name: "sspl-in"
    routing_key: "actuator-req-key"
    username: "sspluser"
    password: "sspl4ever"
    primary_rabbitmq_host: "localhost"
  RABBITMQEGRESSPROCESSOR:
    virtual_host: "SSPL"
    queue_name: "sensor-queue"
    exchange_name: "sspl-out"
    routing_key: "sensor-key"
    ack_queue_name: "sensor-queue"
    ack_exchange_name: "sspl-out"
    ack_routing_key: "sensor-key"
    username: "sspluser"
    password: "sspl4ever"
    message_signature_username: "sspl-ll"
    message_signature_token: "ALOIUD986798df69a8koDISLKJ282983"
    message_signature_expires: "3600"
    iem_route_exchange_name: "sspl-out"
    primary_rabbitmq_host: "localhost"

```

Reply:
@ypise  @mariyappanp https://github.com/Seagate/cortx-prvsnr/pull/592 is taking of this change too. Lets keep one PR only and discard this now